docs/i18n.md

```diff
@@ -1,5 +1,129 @@
+
+## String Freeze
+
+If you're launching or updating a feature, and want to make sure that all new strings are translated before merging,
+add the label [[Status] String Freeze](https://github.com/Automattic/wp-calypso/labels/%5BStatus%5D%20String%20Freeze) to your Pull Request.
+Translatable texts from the `trunk` as well as branches with the `[Status] String Freeze` label are combined and imported into GlotPress.
+In the example below, the translation coverage is 79% and the status check fails.
+
+<img alt="Failed Coverage" src="https://user-images.githubusercontent.com/31164683/187543186-24274733-dcc3-433b-a784-a6a5899b2300.png">
+
+Once the coverage reaches 100%, the status check passes.
+
+## CSS for Multi-Directional Layouts
+
+Use [CSS Logical Properties and Values](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties) when
+building layouts that support both left-to-right (LTR) and right-to-left (RTL) languages.
+While the following is still supported:
+
+```css
+/* LTR-specific code */
+.element {
+  position: absolute;
+  left: 0;
+}
+```
+
+Better:
+
+```css
+/* Will render correctly in LTR or RTL */
+.element {
+  position: absolute;
+  inset-inline-start: 0; /* inset-inline-start evaluates to left in LTR and right in RTL */
+}
+```
+
+Similarly, the [text-align](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align) property supports additional
+values like `start` and `end`. For example, the following is still supported:
+
+```css
+/* LTR-specific code */
+.is-right-align {
+  text-align: right;
+}
+.is-left-align {
+  text-align: left;
+}
+```
+
+Better:
+
+```css
+/* Will render correctly in LTR or RTL */
+.is-right-align {
+  text-align: end; /* end evaluates to right in LTR and left in RTL */
+}
+.is-left-align {
+  text-align: start; /* start evaluates to left in LTR and right in RTL */
+}
+```
+
+Detailed information about the supported properties and values can be found at [css-tricks.com](https://css-tricks.com/building-multi-directional-layouts/).
+
+### Caveats
+
+CSS Logical Properties and Values is a [draft standard](https://drafts.csswg.org/css-logical/) (as of August 2022), and
+some physical CSS properties, like [transform](https://developer.mozilla.org/en-US/docs/Web/CSS/transform), have no logical
+equivalent. [Discussions](https://github.com/w3c/csswg-drafts/issues/1544) and [Proposals](https://github.com/w3c/fxtf-drafts/issues/311)
+exist to standardize the behaviour, but have stalled.
+In such instances, we can rely on the [Webpack RTL Plugin](https://github.com/Automattic/wp-calypso/tree/trunk/packages/webpack-rtl-plugin)
+to transform the content to the correct location on the screen.
+
+## Designing for Internationalization
+
+### Leave room for text expansion
+
+When text is translated from one language to another,
+the length of the text can change. If a text container is too small or has a fixed-sized,
```

Comment:
Looks like a word is missing here, should it be `has a fixed size` or `has a fixed-sized width`?

Reply:
Good catch! It should be `fixed size`.